examples/encfs/README.md

```diff
@@ -109,9 +109,9 @@ lost+found  test.txt
 [!] Closing device...
 ```
 
-The user needs to upload the blob to the previously generated storage container by uploading to the Azure Portal, using [az storage blob upload] (https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-cli), or using [azcopy] (https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json).
+The user needs to upload the blob to the previously generated storage container by uploading to the Azure Portal, using [az storage blob upload] (https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-cli), or using [azcopy] (https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json). When uploading the blob, the type must be specified as a "page blob" for a read-write filesystem.

```

Comment:
When modifying .md files, it is nice to include each sentence on its own line to make it easier to see what is changing between big blocks of text.